CREDITS

```diff
@@ -57,16 +62,14 @@ plugins/setoid_ring
   developed by Benjamin Grégoire (INRIA-Everest, 2005-2006),
   Assia Mahboubi, Laurent Théry (INRIA-Marelle, 2006)
   and Bruno Barras (INRIA LogiCal, 2005-2006),
+plugins/ssreflect
+  developed by Georges Gonthier (Microsoft Research - Inria Joint Centre, 2007-2011),
+  Assia Mahboubi and Enrico Tassi (Inria-Marelle, 2011-now).
```

Comment:
I think putting the Inria team name is too unstable.  Assia was in the Inria joint center, typical, then specfun.  I was in the joint centre, then specfun and now marelle.

Just put Inria

Reply:
OK. What about the ending dates? Shouldn't it be at least 2017 for everyone (including George)?